apps/examples/testcase/tc_main.c

```diff
@@ -56,6 +56,9 @@
 #if defined(CONFIG_EXAMPLES_TESTCASE_MQTT_UTC)
 #define TC_MQTT_STACK 4096
 #endif
+#ifdef CONFIG_EXAMPLES_TESTCASE_AUDIO_UTC
+#define TC_AUDIO_STACK  20480
```

Comment:
20KB is needed??

Reply:
recording needs bunch of memory

Reply:
We checking this now, if it is too huge we will adjust.